packages/ra-ui-materialui/src/button/BulkDeleteButton.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { Component, MouseEvent } from 'react';
```

Comment:
please make sure that the last changes to that component were taken into account